src/pages/fundamentals/typography.hbs

```diff
@@ -50,6 +54,10 @@ styles:
 {{/with}}
 </figure>
 
+<h3 id="typeface-sharp-sans">Firefox Sharp Sans</h3>
+<p>Sharp Sans is a closed lisence font lisenced for use on some Firefox Marketing sites.</p>
+<p>Contact the websites team for more information on how to use this font with Protocol if you belive your use is covered by the lisence.</p>
```

Comment:
SP: "believe" and "license"

I suppose we should probably be more specific about who the websites team is and how to contact us, but I'm not sure how to phrase it... Maybe we can link to the Protocol repo and tell people to open an issue for license questions?